client/src/ui/landingpage/LandingPageHero.tsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image";
+//import { TextCarousel } from "@/ui/TextCarousel";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+
+export function LandingPageHero() {
+  return (
+    <div style={{
```

Comment:
Isn't div wrapper unnecessary? Can we make it into a single div instead that uses grids with two columns instead to make it simpler?

Reply:
I would prefer to keep it this way. This component will most likely only be used once. It works with the structure is has right now